README.md

```diff
@@ -32,6 +32,9 @@ Create a new DNS socket instance. The `options` object includes:
 - `retries` *Number*: Number of total query attempts made during `timeout`. Default: 5.
 - `socket` *Object*: A custom dgram socket. Default: A `'udp4'` socket.
 - `timeout` *Number*: Total timeout in milliseconds after which a `'timeout'` event is emitted. Default: 7500.
+- `maxQueries` *Number*: Each request has an id, this is stored as static sized array. maxQueries is the size of this array, limiting the max number of inflight requests. Default: 10000.
+- `maxRedirects` *Number*: If you query for a single `A` record and get back `CNAME`, the lib will try to follow the chain and resolve the `CNAME` to A. The maximum number of steps is defined by the `maxRedirects`. Default: 0
```

Comment:
I think default should be to follow redirects, maybe with a default maximum value of 50.

Reply:
I didn't want to break existing functionality of the nofollow. 50 is kind of extensive I think, CNAME points to CNAME points to CNAME .... 50 times, more realistic is just a few steps, like CNAME to CNAME to A record.

Reply:
On second thought, I'm ok with having it off by default, to avoid the breaking change.